_includes/locale/en/LC_MESSAGES/keyboards-en.po

```diff
@@ -0,0 +1,85 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+
+# Default English strings for keyboards/index.php
+
+# Page Title
+msgid "Keyboard Search"
+msgstr "Keyboard Search"
+
+# Page Description
+msgid "Keyman Keyboard Search"
+msgstr "Keyman Keyboard Search"
+
+# Keyboard search bar
+msgid "Keyboard search%s"
```

Comment:
Anywhere we have parameters we need to index them. Particularly important if there are multiple parameters in a string.

```suggestion
msgid "Keyboard search%1$s"
```